neo/io/blackrockio.py

```diff
@@ -391,7 +391,7 @@ def __extract_nev_file_spec(self):
         """
         Extract file specification from an .nsx file
```

Comment:
There is also a typo in the docstring here

Reply:
True... fixing